crypto/api/v1/api.go

```diff
@@ -87,6 +92,31 @@ func (signRequest SignJwsRequest) validate() error {
 	return nil
 }
 
+func (signRequest EncryptJweRequest) validate() error {
+	url, err := did.ParseDIDURL(signRequest.To)
+	if err != nil {
+		return err
+	}
+	if url.Empty() {
+		return errors.New("missing to")
```

Comment:
I think `to` is a bit weird in JWE terms (I believe it comes from `JWM`?), so `receiver` might be a better name?

Reply:
Renamed to receiver